src/App.tsx

```diff
@@ -1,14 +1,61 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(false);
+  const [searchFilter, setSeacrhFilter] = useState('');
+  const [completedSearch, setCompletedSearch] = useState('all');
+  const [modalId, setModalId] = useState<undefined | number>(undefined);
+
+  const handleLoadTodos = () => {
+    setLoader(true);
+    getTodos()
+      .then(todosFromServer => {
+        setTodos(todosFromServer);
+      })
+      .then(() => {
+        setLoader(false);
+      });
```

Comment:
```suggestion
      .then(setTodos)
      .then(() => setLoader(false));
```

Reply:
<img width="617" alt="image" src="https://github.com/user-attachments/assets/34ca8866-6876-4d41-b10f-b28640009764">
red minuses - you need to delete it

green plusses - you need to add it